rcl_lifecycle/include/rcl_lifecycle/rcl_lifecycle.h

```diff
@@ -430,6 +430,25 @@ RCL_LIFECYCLE_PUBLIC
 void
 rcl_print_state_machine(const rcl_lifecycle_state_machine_t * state_machine);
 
+/// Print the transition map
+/**
+ * This function will print in the standard output the transition map
+ * of the rcl_lifecycle_state_machine_t struct.
+ *
+ * <hr>
+ * Attribute          | Adherence
+ * ------------------ | -------------
+ * Allocates Memory   | No
+ * Thread-Safe        | No
+ * Uses Atomics       | No
+ * Lock-Free          | Yes
+ *
+ * \param[in] state_machine pointer to the state machine struct to print the transition map
```

Comment:
this is wrong. input is transition map in the state machine structure.

```suggestion
 * \param[in] transition_map pointer to the transition map to print
```